### ISSUE TITLE
Export game shortcuts

### ISSUE THREAD

**Issue by Solaire** [Nutzzz/GLC#9]
The app should allow the user to export multiple games as shortcuts to a given location. Initially, the implementation would only work on single games or a folder (eg. steam) but in the future it could handle selecting multiple games a a time (would need additional features in navigation mode)

**Comment by Nutzzz** [comment]
You may have missed that single-game shortcut to Desktop is already implemented.

**Comment by Solaire** [comment]
Yes, I have missed that. But still in the future we should probably support exporting all icons

**Comment by Nutzzz** [comment]
FYI, this is what I'm currently working on.  As a first step I've got a working file/folder browser using the existing navigation framework.  I expect we may need such a browser for other features (maybe e.g., adding a game from within the app).  I'm trying not to make things too ugly in the process, but it's clear there's going to need to be a refactor at some point.

**Comment by Solaire** [comment]
Okay, that's good.
I am currently working on the SQL stuff and refactoring some of the game classes (and creating a new class for platforms) to use the queries to read/write data.
You are right, we will need to refactor the console code (maybe store all of that in a dedicated TUI class?)

**Comment by Nutzzz** [comment]
This is somewhat complete since [this commit](https://github.com/Solaire/GLC/pull/30/commits/f14b2bd90e84f0d3bd5e5725d7a119be8b5cb488) was pulled, though I'd like to add the ability to manually type a path into the folder browser.
